src/OrchardCore.Modules/OrchardCore.Media.Azure/CreateMediaBlobContainerEvent.cs

```diff
@@ -77,13 +77,13 @@ public override async Task RemovingAsync(ShellRemovingContext context)
                 if (!response.Value)
                 {
                     _logger.LogError("Unable to remove the Azure Media Storage Container {ContainerName}.", _options.ContainerName);
-                    context.LocalizedErrorMessage = S["Unable to remove the Azure Media Storage Container '{0}'.", _options.ContainerName];
+                    context.ErrorMessage = S["Unable to remove the Azure Media Storage Container '{0}'.", _options.ContainerName];
```

Comment:
Elastic also needs to support removal: https://github.com/OrchardCMS/OrchardCore/pull/11890#issuecomment-1250382438

Reply:
No luck, elastic was merged first ;) I will start on this at the end and see if I need help from @Skrypt, at least his review.

Reply:
Okay done, would be good to reviewed by @Skrypt, see in the existing `ElasticIndexInitializerService.cs` which was already a `ModularTenantEvents`.